app/Admin/Theme_Settings.php

```diff
@@ -9,7 +9,7 @@ class Theme_Settings extends \JustCoded\WP\Framework\Admin\Theme_Settings {
 	 * Create admin page for theme settings
 	 */
 	public function init() {
-		$panel = static::titan_instance()->createContainer( array(
+		$panel = self::$panel = static::titan_instance()->createContainer( array(
```

Comment:
use static::$panel.

I think it's better to not use another $panel variable in case you already have static variable